src/components/App/App.js

```diff
@@ -40,28 +40,30 @@ const App = () => {
       <Rules rulesOpen={areRulesOpen} />
       <Header />
       <div className="utilities">
+        <DiceRoller />
         <IconButton
-          name="Open rules"
+          name="Rules"
           imgSrc={rulesIconSrc}
           onClickFunction={() => toggleRules()}
```

Comment:
I know you didn't change anything here, but generally, when building components, using the same properties as the react defaults is preferred. A developer unfamiliar with `IconButton` would assume this prop would be named `onClick` but it's not. This reduces cognitive load